LinqToRest/QueryProviderBuilderException.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+
+namespace Messerli.LinqToRest
+{
+    public class QueryProviderBuilderException : Exception
+    {
+        public QueryProviderBuilderException()
+        {
+        }
+
+        public QueryProviderBuilderException(string message)
+            : base(message)
+        {
+        }
+
+        public QueryProviderBuilderException(string message, Exception inner)
+            : base(message, inner)
+        {
+        }
+    }
+}
```

Comment:
Terminating newline

Reply:
Addressed in https://github.com/messerli-informatik-ag/linq-to-rest/pull/17/commits/4389e5c27fad7d25a9309580a00d534b44f25f2f